setup-node.sh

```diff
@@ -343,9 +343,12 @@ setup_seda() {
         cosmovisor run config set app telemetry.prometheus-retention-time 60
     fi
 
-    # Disable seda signer as we're not spinning up a validator
-    cosmovisor run config set app seda.enable-seda-signer false
-    
+    # Disable seda signer as we're not spinning up a validator if the version is 1.0.0 or higher
+    if [[ $SEDA_VERSION == v1.0.0* ]]; then
```

Comment:
Maybe we can make it so we disable it only when the version is not v1*? Cause I feel the current change will fail if we get to v1.1.0 right?